client/state/sites/constants.js

```diff
@@ -38,9 +38,10 @@ export const SITE_REQUEST_OPTIONS = [
 	'is_mapped_domain',
 	'is_redirect',
 	'is_wpcom_store',
-  'woocommerce_is_active',
+	'woocommerce_is_active',
 	'jetpack_version',
 	'main_network_site',
+	'onboarding_checklist_initialized',
```

Comment:
Do we really need this off the site object? Please be careful about bloating the site object further. We do have the option of adding additional endpoints if necessary.

Do you mind expanding on what this prop means? It still looks like the client is performing a number of checks in addition to looking at this prop.

Reply:
It basically means that the checklist has been initialized on the server side. I think that in practice it should be pretty redundant with the site age check already happening in the reducer so I'll just eliminate it as I'd like to get this out the door.

Reply:
Thanks for the extra context @mattwiebe !